packages/create-webstone-app/templates/plugin/structure/packages/cli/package.json

```diff
@@ -7,7 +7,8 @@
 		"clean": "rimraf build",
 		"copy-templates": "copyfiles ./src/templates/* ./build/templates",
 		"format": "prettier --plugin-search-dir . --write .",
-		"build": "tsc -p tsconfig.json",
+		"build": "npm run clean && run-s build:cli copy-templates",
```

Comment:
forgot to copy the templates during the build step

Reply:
Moved this to #283.